lib/ccd/templates/problem_concern_act_condition_template.rb

```diff
@@ -10,10 +10,9 @@ def self.included(base)
       constraint 'mood_code', {:cardinality=>"1..1", :value=>"EVN"}
       
       # SHALL contain exactly one [1..1] templateId (CONF:16772) such that it
-      constraint 'template_id', {:cardinality=>"1..1"}
-      
       # SHALL contain exactly one [1..1] @root="2.16.840.1.113883.10.20.22.4.3" (CONF:16773).
-      constraint 'template_id.root', {:cardinality=>"1..1", :value=>"2.16.840.1.113883.10.20.22.4.3"}
+      constraint 'template_id', {:cardinality=>"1..1", :value=>{:root => "2.16.840.1.113883.10.20.22.4.3"}}
+      constraint 'template_id', {:cardinality=>"1..1", :value=>{:root => "2.16.840.1.113883.10.20.22.4.3", :extension => "2015-08-01"}}
```

Comment:
It's been a while since I've looked at this library.  If we use the same constraint twice, does one overwrite the other?

Reply:
It generates 2 `templateId` tags which is what's in the official ccds  https://github.com/onc-healthit/2015-certification-ccda-testdata/blob/master/Receiver%20SUT%20Test%20Data/170.315_b2_CIRI_Amb/170.315_b2_ciri__r21_sample1_ccd_v11.xml#L25-L30

```xml
<!-- US Realm Header ID-->
<templateId root="2.16.840.1.113883.10.20.22.1.1" extension="2015-08-01"/>
<templateId root="2.16.840.1.113883.10.20.22.1.1"/>
<!-- CCD template ID-->
<templateId root="2.16.840.1.113883.10.20.22.1.2" extension="2015-08-01"/>
<templateId root="2.16.840.1.113883.10.20.22.1.2"/>
```